### ISSUE TITLE
Tooltip can be hovered

### ISSUE THREAD

**Issue by trybern** [FHIDev/Fhi.Designsystem#229]
### Description

Tooltips should be possible to hover, [according to success criterion 1.4.13 of WCAG 2.1.](https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html) Our current tooltip is dismissed when users try to hover it.

### Acceptance Criteria

- Tooltip persists when the user moves their cursor from the trigger target to the tooltip itself.
